### DISCUSSION TITLE
Welcome to Natural-JS Discussions!

### DISCUSSION THREAD

**Discussion by bbalganjjm** [bbalganjjm/natural_js#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wall72** [comment]
OK. I'm in. 👍

**Reply by bbalganjjm** [reply]
깃헙에 좋은게 생겼어요. 게시판이 필요했는데...
